libs/dh/metering-point/feature-search/src/routes.ts

```diff
@@ -49,7 +49,9 @@ const marketRolesWithDataAccess = [
 export const dhMeteringPointRoutes: Routes = [
   {
     path: '',
-    canActivate: [PermissionGuard(['metering-point:search'])],
+    canActivate: [
+      PermissionGuard(['metering-point:search'], `/${getPath<BasePaths>('message-archive')}`),
```

Comment:
Suggestion: Maybe we should add / in permission.guard.ts file, so the next developer doesnt forget to do that?